test/src/tpm/tpm2_interface_test.c

```diff
@@ -749,3 +761,104 @@ void test_rollNonces(void)
   CU_ASSERT(rollNonces(&session, new) != 0);
 
 }
+
+
+
+//----------------------------------------------------------------------------
+// test_unseal_apply_policy
+//----------------------------------------------------------------------------
+void test_unseal_apply_policy(void)
```

Comment:
This test assumes that the default hash algorithm is SHA-256. If this is ever changed, I believe that these tests will break. Perhaps we should put a conditional around the current tests to only run them if the expected default value is set and warn that tests were skipped if the default is something we didn't expect.

```c
if (KMYTH_HASH_ALG == TPM2_ALG_SHA256)
{
    [tests]
}
else
{
    [warning]
}
```

Reply:
Added check